etc/picongpu/cori-nersc/a100_picongpu.profile.example

```diff
@@ -0,0 +1,77 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ################################# (edit the following lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+export proj="<yourProject>"
```

Comment:
Hackathon project name: likely `m3502`

Reply:
Are we allowed to put it in public?

Reply:
This is not sensitive info. A user has to be added to a project to get membership.

I would not commit it to the docs though, because that won't help people. I just add it here for you.